### DISCUSSION TITLE
Web Search broken

### DISCUSSION THREAD

**Discussion by SouthFresh** [open-webui/open-webui#7105] [Issues]
# Bug Report

Note: Please remove the notes above when submitting your post. Thank you for your understanding and support!

---

## Installation Method

Docker Compose including searchxng docker compose file.

## Environment

- **Open WebUI Version:** v0.4.1(latest)
- **Ollama (if applicable):** v0.4.2

- **Operating System:** Ubuntu 24.04 LTS
- **Browser (if applicable):** Multiple

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Expect Open-WebUI client to establish a connection to the running Searxng instance and retrieve search results.

## Actual Behavior:

Error is received and no search results are provided.

## Description

**Bug Summary:**
Web search appears to universally fail across all queries. The error message states that max retries exceeded caussed by "
"NewConnectionError('<urllib3.connection.HTTPConnection object..." included in logs.

## Reproduction Details

**Steps to Reproduce:**
1. Create new chat
2. Enable web search
3. Ask selected LLM to search web for x

## Logs and Screenshots

**Browser Console Logs:**
[console-export-2024-11-20_11-15-15.txt](https://github.com/user-attachments/files/17833598/console-export-2024-11-20_11-15-15.txt)

**Docker Container Logs:**
[_open-webui_logs.txt](https://github.com/user-attachments/files/17833612/_open-webui_logs.txt)
[_searxng_logs.txt](https://github.com/user-attachments/files/17833625/_searxng_logs.txt)

**Screenshots/Screen Recordings (if applicable):**
![image](https://github.com/user-attachments/assets/793b430e-ee4d-4412-8222-54bf3f363e7e)
![image](https://github.com/user-attachments/assets/9a86c96f-3b70-4f7f-aa6f-31621ed004ee)
![image](https://github.com/user-attachments/assets/79a80363-b1e4-4e3c-b1f3-11602479cfc9)

**Comment by tjbck** [comment]
Related: #7096

Help from the community wanted here.

**Reply by SouthFresh** [reply]
Sorry about the miscategorization, I originally filed this as a bug because it arose with the .4 update, without any changes on SearXng

**Comment by codingstube** [comment]
After updating to version 0.4 < I got the same issue with searxng or duckduckgo. No searchresult found.

**Comment by SouthFresh** [comment]
Reconfigure and this issue is gone. Sorry for the waste of time.
